CONTRIBUTING.md

```diff
@@ -21,17 +21,34 @@ Behat requires a Pantheon site. Once you've created the site, you'll need [insta
 
 ## Workflow
 
-The `develop` branch is the development branch which means it contains the next version to be released. `main` contains the corresponding stable development version. Always work on the `develop` branch and open up PRs against `develop`.
+Development and releases are structured around two branches, `develop` and `main`. The `develop` branch is the default branch for the repository, and is the source and destination for feature branches.
+
+We prefer to squash commits (i.e. avoid merge PRs) from a feature branch into `develop` when merging, and to include the PR # in the commit message. PRs to `develop` should also include any relevent updates to the changelog in readme.txt. For example, if a feature constitutes a minor or major version bump, that version update should be discussed and made as part of approving and merging the feature into `develop`.
```

Comment:
Is it possible (and if it is have we done this) to set `Squash and merge` as a repository default?

Reply:
we don't want it as a default because we want to `Rebase and Merge` into `main`: https://github.com/pantheon-systems/wp-native-php-sessions/pull/252/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R41

I'm not sure if we can make branch-specific merge rules.